### DISCUSSION TITLE
How to force xTB to create the recovery file for MD

### DISCUSSION THREAD

**Discussion by icamps** [grimme-lab/xtb#759] [Q&A] [answered]
Hello,

Is it possible to force xTB in such a way to always write the recovery file for Molecular Dynamic (MD) calculations?

I am running MD for small systems, and xTB create the `mdrestart` file but for large files, it doesn't. The large files are running in a HPC center with time limitation and the calculations for large system stop due to time.

Best regards,

Camps

**Comment by MtoLStoN** [comment, marked as answer]
At the moment, xTB only writes ``mdrestart`` files at the end of an MD calculation. In #760 I added the option to force the writing of ``mdrestart`` files when writing the ``scoord`` files. You can enable this by adding 
```
forcewrrestart=true
```
in your extended xtb input file in the ``$md`` block after the PR is merged (which will probably be soon).
